Makefile

```
@@ -249,6 +250,21 @@ generate-fakes: ## Generate fakes for writing unit tests
 verify: gomod fmt ## Run all verification scripts
 	./hack/check/check-dirty.sh
 
+## --------------------------------------
+## Generators
+## --------------------------------------
+
+CONTROLLER_GEN_SRC ?= "./..."
+
+generate-controller-code: $(CONTROLLER_GEN)  ## Generate code via controller-gen
+	$(CONTROLLER_GEN) $(GENERATOR) object:headerFile="$(ROOT_DIR)/hack/boilerplate.go.txt",year=$(shell date +%Y) paths="$(CONTROLLER_GEN_SRC)" $(OPTIONS)
+	$(MAKE) fmt
+
+generate-manifests:
+	$(MAKE) generate-controller-code GENERATOR=crd OPTIONS="output:crd:artifacts:config=$(ROOT_DIR)/apis/config/crd/bases" CONTROLLER_GEN_SRC=$(CONTROLLER_GEN_SRC)
+
+generate: generate-controller-code generate-manifests
```

nit: Add help text to make it appear in the `make help` output.

```suggestion
generate: generate-controller-code generate-manifests ## Generate controller code and manifests e.g. CRD etc.
```

---

updated. thanks!